projects/plugins/boost/app/modules/cache/Page_Cache.php

```diff
@@ -34,20 +34,24 @@ public function __construct() {
 		register_uninstall_hook( JETPACK_BOOST_PATH, array( Page_Cache_Setup::class, 'uninstall' ) );
 	}
 
-	public function setup() {}
+	public function setup() {
+		Garbage_Collection::init();
+	}
 
 	/**
 	 * Runs the setup when the feature is activated.
 	 */
 	public static function activate() {
 		Page_Cache_Setup::run_setup();
+		Garbage_Collection::install();
 	}
 
 	/**
 	 * Runs cleanup when the feature is deactivated.
 	 */
 	public static function deactivate() {
 		Page_Cache_Setup::deactivate();
+		Garbage_Collection::uninstall();
```

Comment:
Setup calls init
activate calls install
deactivate calls uninstall

I feel like we should keep the naming consistent.